meshtastic/__main__.py

```
@@ -808,18 +813,23 @@ def onConnected(interface: MeshInterface) -> None:
                 )
+            if args.dest not in {BROADCAST_ADDR, LOCAL_ADDR}:
+                _cli_exit(
+                    "Error: OTA update only supports the directly connected local node; omit --dest or use --dest ^local."
+                )
+            ota_dest = BROADCAST_ADDR if args.dest == LOCAL_ADDR else args.dest
 
             ota = meshtastic.ota.ESP32WiFiOTA(args.ota_update, interface.hostname)
 
             print(f"Triggering OTA update on {interface.hostname}...")
-            interface.getNode(args.dest, False, **getNode_kwargs).startOTA(
+            interface.getNode(ota_dest, False, **getNode_kwargs).startOTA(
```

Ruff (0.15.4)

[error] Boolean positional value in function call

(FBT003)